src/validator/Validator.js

```diff
@@ -97,10 +97,10 @@ Validator.prototype.addRule = function(rule) {
     return this;
 };
 
-Validator.prototype.addRules = function(rules) {
-    var _this = this;
-    _.each(rules, function(rule) {
-        _this.addRule(rule);
+Validator.prototype.addRules = function (rules) {
+    var self = this;
+    _.each(rules, function (rule) {
+        self.addRule(rule);
```

Comment:
**core:** While updating it, better to go further and use a lambda

**suggestion:**
```suggestion
    _.forEach(rules, rule => this.addRule(rule));
```

Reply:
Thanks, done